state/state.go

```diff
@@ -7,9 +7,16 @@ import (
 	"context"
 
 	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/set"
 	"github.com/ava-labs/avalanchego/x/merkledb"
 )
 
+const (
```

Comment:
Could you add one more value here for `Allocate` (fine to leave as a `TODO`)? -> This won't help with parallelization but will help with fee calculation (if you don't allow `Allocate`, can charge less fee).